### PR TITLE
SNO Dshboard initial api endpoint added

### PR DESCRIPTION
**Pull request by crawter**
What: 
all static resources are only for test purpose
Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: storagenode/console/consoleserver/server.go [hunk 1]
```diff
@@ -88,3 +105,76 @@ func (s *Server) Close() error {
...
+
+	walletAddress := s.service.GetWalletAddress(ctx)
+
+	versionInfo := s.service.GetVersion(ctx)
+
+	err = s.service.CheckVersion(ctx)
+	if err != nil {
+		s.log.Error("can not check latest storagenode version", zap.Error(err))
+		response.Error = "can not check latest storagenode version"
+		return
```
Comment (rikysya) [inline storagenode/console/consoleserver/server.go]: If you return here then it makes next check irrelevant

File: storagenode/console/consoleserver/server.go [hunk 2]
```diff
@@ -88,3 +105,76 @@ func (s *Server) Close() error {
...
+	}
+
+	var isLastVersion = err == nil
```
Comment (rikysya) [inline storagenode/console/consoleserver/server.go]: why var?

File: storagenode/console/consoleserver/server.go [hunk 3]
```diff
@@ -88,3 +105,76 @@ func (s *Server) Close() error {
...
+	var response struct {
+		Bandwidth     console.BandwidthInfo `json:"bandwidth"`
+		DiskSpace     console.DiskSpaceInfo `json:"diskSpace"`
+		WalletAddress string                `json:"walletAddress"`
+		VersionInfo   version.Info          `json:"versionInfo"`
+		IsLastVersion bool                  `json:"isLastVersion"`
+		Uptime        time.Duration         `json:"uptime"`
+		NodeID        string                `json:"nodeId"`
+		Satellites    storj.NodeIDList      `json:"satellites"`
+		Error         string                `json:"error,omitempty"`
```
Comment (rikysya) [inline storagenode/console/consoleserver/server.go]: Why error mixed with other data?

File: storagenode/console/consoleserver/server.go (hunk 1 above)
Comment (rikysya) [reply storagenode/console/consoleserver/server.go]: I think we need to provide the latest version as well

File: storagenode/console/consoleserver/server.go [hunk 4]
```diff
@@ -88,3 +105,133 @@ func (s *Server) Close() error {
...
+
+	satelliteIDParam := req.URL.Query().Get("satelliteId")
+	s.log.Info("param: " + satelliteIDParam)
+	satelliteID, err := s.parseSatelliteIDParam(satelliteIDParam)
+	if satelliteID != nil {
+		s.log.Info("ID: " + satelliteID.String())
+	}
+	if err != nil {
+		s.log.Error("satellite id is not valid", zap.Error(err))
+		response.Error = "satellite id is not valid"
```
Comment (egonelbre) [inline storagenode/console/consoleserver/server.go]: This is not setting the proper HTTP status code.

File: storagenode/console/consoleserver/server.go [hunk 5]
```diff
@@ -88,3 +105,133 @@ func (s *Server) Close() error {
...
+		return
+	}
+
+	satellites, err := s.service.GetSatellites(ctx)
+	if satellites == nil {
+		s.log.Info("SATELLITES ARE NIL")
+	}
+	if err != nil {
+		s.log.Error("can not get satellites list", zap.Error(err))
+		response.Error = "can not get satellites list"
```
Comment (egonelbre) [inline storagenode/console/consoleserver/server.go]: This is not setting the proper HTTP status code.

File: storagenode/console/consoleserver/server.go [hunk 6]
```diff
@@ -88,3 +105,133 @@ func (s *Server) Close() error {
...
+		return
+	}
+
+	if satelliteID != nil {
+		if err = s.checkSatelliteID(satellites, *satelliteID); err != nil {
+			s.log.Error(err.Error())
+			response.Error = err.Error()
```
Comment (egonelbre) [inline storagenode/console/consoleserver/server.go]: This is not setting the proper HTTP status code.

**Pull request merged**